setup/jest.config.js

```diff
@@ -305,6 +305,8 @@ module.exports = {
     'src/modules/account/hooks/useCurrentAccount.js',
     'src/modules/transaction/store/actions.js',
     'src/modules/transaction/hooks/useCreateTransaction.js',
+    'src/modules/common/hooks/useOutsideClick.js',
+    'src/modules/common/hooks/useCustomQuery.js',
```

Comment:
was there a reason for excluding coverage checks for `useOutsideClick` and `useCustomQuery` @Balanced02